### ISSUE TITLE
Add `HeadNode:Iam:S3Access` option to the wizard

### ISSUE THREAD

**Issue by stephenmsachs** [aws-samples/pcluster-manager#269]
While `HeadNode:CustomActions:OnNodeStart:Script` can be pointed to an s3 location in the wizard, there is currently no mechanism to add the pre-requisite `HeadNode:Iam:S3Access`. This results in an error trying to use the s3 location as script. `S3Access` can be granted by hand in the yaml file, but you need to know a few things about pcluster to know where to add the optiion.

**Comment by mendaomn** [comment]
Good point, thank you. We'll investigate the issue and see if there's a better approach

**Comment by eantonin** [comment]
We reproduced the issue and put it in our roadmap, cannot give a precise ECD yet

**Comment by sean-smith** [comment]
Hi Stephen another workaround until we fix the issue is to add the `AmazonS3ReadOnlyAccess` managed policy to the cluster, this is what we do in the weather workshop. It's allowed without any modification to pcluster manager:

https://weather.hpcworkshops.com/template/cluster-config.yaml

**Comment by stephenmsachs** [comment]
Yes, thanks Sean. That's what I am doing for now.
